exercises/J-functions/exercise.js

```diff
@@ -1,5 +1,7 @@
 function halve(number) {
   // complete the function here
+   var result = (number / 2);
+   return result; 
```

Comment:
To make this a tiny bit better you could write

```
return number/2
```

Since you can return directly after a statement